frontend/src/pages/detail/Detail.tsx

```diff
@@ -53,10 +91,55 @@ function Detail() {
 export default Detail;
 
 const StyledContainer = styled.div`
+  padding: 0 2rem;
+`;
+
+const StyledMentorInfoWrapper = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
   gap: 2.4rem;
+`;
 
-  padding: 0 2rem 2rem;
+const StyledTapWrapper = styled.div`
+  position: relative;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  padding: 1rem;
+`;
+
+const StyledTap = styled.p<{ selected?: boolean }>`
+  width: 50%;
+  cursor: pointer;
+  text-align: center;
+
+  ${({ theme }) => theme.TYPOGRAPHY.B2_B};
+`;
+
+const StyledTapIndicator = styled.div<{ selected?: 'detail' | 'review' }>`
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 50%;
+  height: 1px;
+  background-color: ${({ theme }) => theme.SYSTEM.MAIN500};
+  transition: transform 0.2s ease-in-out;
+  z-index: 0;
+
+  transform: ${({ selected }) =>
+    selected === 'detail' ? 'translateX(0%)' : 'translateX(100%)'};
+`;
+
+const StyledContentWrapper = styled.div`
+  display: flex;
+  width: 100%;
+  transition: transform 0.3s ease-in-out;
+  padding-top: 2rem;
+`;
+
+const StyledDetailWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 2.4rem;
```

Comment:
p3: 혹시 상세 보기 영역이 반으로 잡혀서 그런데 여기에 width:100% 가 추가되면 어떨까? 🤔

<img width="601" height="413" alt="스크린샷 2025-08-12 오후 4 02 43" src="https://github.com/user-attachments/assets/20badc3a-8537-40f2-a9f1-404c7a2ab44b" />

```suggestion
const StyledDetailWrapper = styled.div`
  display: flex;
  flex-direction: column;
  gap: 2.4rem;
  width: 100%
```

Reply:
이건 스타일이 누락됐네... 수정할때 스타일이 덜제거되거나 누락된게 있는거 같아! 꼼꼼한 리뷰 고마워~~

Reply:
[style: 누란된 width: 100% 스타일 추가](https://github.com/woowacourse-teams/2025-Fit-toring/pull/328/commits/67a293f6991173d9e957cf3259b6d02c4a6ba2ea) https://github.com/woowacourse-teams/2025-Fit-toring/issues/309